debian/rules

```diff
@@ -14,20 +14,8 @@ export PATH:=/usr/lib/go-1.6/bin:${PATH}
 
 include /etc/os-release
 
-SYSTEMD_UNITS_DESTDIR=
-ifeq (${VERSION_ID},"14.04") 
-	# We are relying on a deputy systemd setup for trusty,
-	# in which systemd does not run as PID 1. To solve the
-	# problem of services shipping systemd units and upstart jobs
-	# being started twice, we altered systemd on trusty to ignore
-	# /lib/systemd/system and instead consider only selected units from
-	# /lib/systemd/upstart.
-	SYSTEMD_UNITS_DESTDIR="lib/systemd/upstart/"
-	# make sure that trusty's golang-1.6 is picked up correctly.
-	export PATH:=/usr/lib/go-1.6/bin:${PATH}
-else
-	SYSTEMD_UNITS_DESTDIR="lib/systemd/system/"
-endif
+# this is overriden in the ubuntu/14.04 release branch
+SYSTEMD_UNITS_DESTDIR="lib/systemd/system/"
```

Comment:
I think we still want the adjustment to PATH here. Otherwise, we will probably run into package building issue or build with the wrong go toolchain.